### PR TITLE
Fix: retry db transaction if we get a serialization error when persisting anchors

### PR DESCRIPTION
**Pull request by stephhuynh18**
Fixes error "could not serialize access due to concurrent update". This is the only place that does not catch this error and retry

### CODE REVIEW

**Review by gvelez17 — APPROVED** [review]
LGTM

**Review by smrz2001 — APPROVED** [review]
🥳

File: src/services/anchor-service.ts [hunk 1]
```diff
@@ -546,40 +547,46 @@ export class AnchorService {
...
-      completed.publishStats(METRIC_NAMES.CREATED_SUCCESS_MS)
-      Metrics.count(METRIC_NAMES.ACCEPTED_REQUESTS, acceptedRequests.length)
-
-      return persistedAnchorsCount
-    } catch (err) {
-      await trx.rollback()
-      throw err
-    }
+      .catch(async (err) => {
+        if (err?.code === REPEATED_READ_SERIALIZATION_ERROR) {
```
Comment (stbrody) [inline src/services/anchor-service.ts:582]: should we still log the error here?

Comment (stephhuynh18) [reply src/services/anchor-service.ts:582]: I can add a log sure

yes the rollback is handled automatically

**Pull request merged**